docker/ccf.proxy.yaml

```diff
@@ -0,0 +1,39 @@
+# Copyright 2017 Intel Corporation
+#
```

Comment:
copyrights should be updated on the new files (2022)

Reply:
thanks, will fix